hack/dockerfile/install/rootlesskit.installer

```diff
@@ -21,7 +21,7 @@ install_rootlesskit() {
 
 install_rootlesskit_dynamic() {
 	export ROOTLESSKIT_LDFLAGS="-linkmode=external" install_rootlesskit
-	export BUILD_MODE="-buildmode=pie"
+	export BUILD_MODE=${GO_BUILDMODE}
```

Comment:
Can we use `BUILD_MODE` everywhere instead of introducing a new variable `GO_BUILDMODE`

Reply:
I think `GO_BUILDMODE` is more specific and accurate. That change `GO_BUILDMODE` to `BUILD_MODE`  is also OK.

Reply:
Yeah, IMO this should go the other way and update everything to use the more specific `GO_BUILDMODE` name.